wp-content/themes/bzLLKit/functions.php

```diff
@@ -666,6 +666,11 @@ function bz_meta_boxes( $meta_boxes ) {
 				'name' => __( 'Fellows&#39;s Pre-work<ul><li>Please use bullet list.</li><li>To show the cohort&#39;s answers from the module use this shortcode (at least one magic field ID is required, use commas to add more): <code>[cohort-answers="place-magic-field-id-here, optioanlly-another-one-here"]</code></li></ul>', 'bz' ),
 				'type' => 'wysiwyg',
 			),
+			array(
+				'id'   => 'bz_kit_prework_header',
+				'name' => __( 'Use custom header for Pre-work</br>Default: "Fellows Pre-work"', 'bz'),
```

Comment:
Does this need an extra quote to mactch `Fellows'` [below](https://github.com/beyond-z/kits/pull/115/files#diff-97768072f76fac9809f2a6daae266e2cR229)?

Reply:
Good catch! I will make it match.